Runtime/safe-ds/safeds/data/tabular/_table.py

```diff
@@ -163,9 +164,14 @@ def from_rows(rows: list[Row]) -> Table:
 
         Raises
         ------
+        MissingDataError
+            If an empty list without a row is given.
```

Comment:
```suggestion
            If an empty list is given.
```

This comment seems a bit redundant